src/app/account-details/personal-details.component.ts

```diff
@@ -158,10 +160,20 @@ export class PersonalDetailsComponent {
             .post('/rest/v1/account/details', updates)
             .pipe(map((res: HttpResponse<any>) => res['result']))
             .subscribe((details) => {
-                this.details.next(details);
+                if(details && details.email_alternative) {
```

Comment:
I'll mention this once just in case you aren't aware. I'm fine with the code as is. Optional chaining feature allows you to write `detail?.email_alternative` instead of the `details && details...`

Reply:
Vaguely aware, I lose track of which features occur in which eras of ecma/node/typescript/dart (I code the latter to, v confusing!). I'd prefer if we start adopting newer syntax we do it deliberately, whole classes at a time